docs/source/models/concept_models/vivarium_moud/concept_model.rst

```diff
@@ -0,0 +1,177 @@
+
+.. _2021_concept_model_vivarium_moud:
+
+=====================================================
+Medications for Opioid Use Disorder (MOUD) Simulation
+=====================================================
+
+.. contents::
+  :local:
+  :depth: 1
+
+
+
+1 Overview
+++++++++++
+
+This document outlines the concept model for simulating Medications for Opioid Use Disorder (MOUD) interventions. The core of this model is a state-transition model representing opioid use disorder states and treatment dynamics.
+
+2 Modeling aims and objectives
+++++++++++++++++++++++++++++++
+
+TK - Objectives and goals of the MOUD modeling effort
+
+TK - quarters submodel that captures living in a private residence, living unhoused, and incarceration
+
+3 Concept model and submodels
++++++++++++++++++++++++++++++
+
+3.1 Core Disease Model
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. graphviz::
+
+  digraph moud_model {
+      rankdir = LR;
+      susceptible [label="susceptible"]
+      with_condition [label="with condition"]
+      on_treatment [label="on treatment"]
+
+      susceptible -> with_condition [label="i"]
+      with_condition -> susceptible [label="r"]
+      with_condition -> on_treatment [label="ti"] 
+      on_treatment -> with_condition [label="tf"]
+      on_treatment -> susceptible [label="ts"]
+  }
+
+
+The core disease model represents opioid use disorder as a state machine with three states:
+
+.. list-table:: State Definitions
+  :widths: 7 20
+  :header-rows: 1
+
+  * - State
+    - Definition
+  * - susceptible
+    - Individual does not have opioid use disorder
+  * - with condition
+    - Individual has opioid use disorder but is not receiving medication treatment
+  * - on treatment
+    - Individual has opioid use disorder and is receiving medication treatment (MOUD)
+
+.. list-table:: Transition Rate Definitions
+  :widths: 1 5 20
+  :header-rows: 1
+
+  * - Symbol
+    - Name
+    - Definition
+  * - i
+    - incidence rate
+    - Rate at which individuals develop opioid use disorder
+  * - r
+    - remission rate  
+    - Rate at which individuals with untreated OUD naturally recover
+  * - ti
+    - treatment initiation rate
+    - Rate at which individuals with OUD begin medication treatment
+  * - tf
+    - treatment failure rate
+    - Rate at which individuals on MOUD discontinue or fail treatment
+  * - ts
+    - treatment success rate
+    - Rate at which individuals on MOUD achieve sustained recovery/remission
+    
+Key features of the model include:
+
+- The "with_condition" state can be used as a risk exposure
+- Treatment state has no excess mortality or disability weight relative to baseline
+- State membership is determined by prevalence data and treatment ratios loaded from artifact (designed to be internally consistent and to match GBD 2021 estimates)
+
+
+4 Data Notes
+++++++++++++
+
+The model requires the following data streams:
+
+- Overall OUD prevalence
+- Treatment coverage ratios
+- Transition rates:
+  - Treatment initiation
```

Comment:
This nested list didn't work

Reply:
You probably need a blank line above the first item in the sublist